sdks/java/core/pom.xml

```diff
@@ -122,6 +122,12 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
+          <excludes>
+            <exclude>**/package-info.java</exclude>
```

Comment:
The new excludes section overrides the global one, so I had to repeat here. If you have a maven tip to avoid it, I'm happy to accept.